kpatch/kpatch

```diff
@@ -163,7 +163,16 @@ load_module () {
 	fi
 
 	echo "loading patch module: $1"
-	insmod "$1" "$2"
+	for i in `seq 5`; do
+		if [[ $i != 1 ]]; then
+			echo "retrying..."
+			sleep 2
+		fi
+		insmod "$1" "$2" && break
```

Comment:
Oops, yeah.  This is some ugly code.  These functions should really be using local variables to deal with the function arguments.